.circleci/config.yml

```diff
@@ -310,6 +310,20 @@ references:
         demisto-sdk create-id-set -o ./Tests/id_set.json
         cp ./Tests/id_set.json $CIRCLE_ARTIFACTS
 
+  merge_private_and_public_id_sets: &merge_private_and_public_id_sets
+    run:
+      name: Merge private and public ID sets
+      when: always
+      command: |
+        GCS_PATH=$(mktemp)
+        ZIP_FOLDER=$(mktemp -d)
+        echo $GCS_MARKET_KEY > $GCS_PATH
+        python3 Tests/Marketplace/download_private_id_set.py -b 'marketplace-dist-private-test' -s $GCS_PATH
+        rm $GCS_PATH
+
+        demisto-sdk merge-id-sets -i1 /home/circleci/project/artifacts/private_id_set.json -i2 ./Tests/id_set.json -o ./Tests/private_and_public_id_sets.json
```

Comment:
please change the i1,i2 arguments to more informative names
also, put the paths in vars and then use them

Reply:
1 - This is how it is in sdk it is not my choice
2 - done